src/App/App.test.tsx

```diff
@@ -25,39 +108,40 @@ describe('App', () => {
 	});
 
 	it('Should be able to login, then be directed to the All Cocktails Page', async () => {
-		const { getByLabelText, getByText, getByPlaceholderText, getAllByText } = render(<MemoryRouter><App /></MemoryRouter>);
+		const { getByLabelText, getByText, getByPlaceholderText, getAllByText } = render(
+			<MemoryRouter><App /></MemoryRouter>
+		);
+		
+		await waitFor(() => expect(getAllCocktails).toHaveBeenCalled())
```

Comment:
So smart! I should have known I needed that when I was first working on it. Great job!